lib/OpenQA/Controller/Admin/Table.pm

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2014 SUSE Linux Products GmbH
```

Comment:
I just moved the file, but it was indeed very likely written in 2015 - ask Vlada :)